odl/contrib/datasets/ct/mayo.py

```diff
@@ -141,6 +136,11 @@ def load_projections(folder, indices=None):
     # This increases the resolution of the reconstructions.
     angles = angles - offset_angular
 
+    # We correct for the mean offset due to ffz in the detector.
```

Comment:
What is ffz?

Reply:
Flying focal spot (I guess we all know why they use "z" instead. Anyway, I'll write it out.